app/subscriber/src/components/box/Box.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { ReactNode } from 'react';
+import { Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IBoxProps extends Omit<React.HtmlHTMLAttributes<HTMLDivElement>, 'title'> {
+  icon?: ReactNode;
+  title?: ReactNode;
+  canShrink?: boolean;
+  expand?: boolean;
+  onExpand?: (expand: boolean) => boolean;
+  actions?: ReactNode;
+}
+
+export const Box = React.forwardRef<HTMLDivElement, IBoxProps>(
```

Comment:
A new shared component.  Probably will move this to `tno-core` when I update the Editor app report forms.

Reply:
![image](https://github.com/bcgov/tno/assets/3180256/a87bba8b-f113-446f-a617-b05234eb53f1)